package.yml

```
@@ -54,9 +58,20 @@ pages:
                 itemClass: pull-right
                 perm: neues[docs]
 
+yform:
+    neues/entry:
+        table_name: FriendsOfRedaxo\Neues\Entry
+    neues/category:
+        table_name: FriendsOfRedaxo\Neues\Category
+    neues/author:
+        table_name: FriendsOfRedaxo\Neues\Author
+    neues/lang:
+        table_name: FriendsOfRedaxo\Neues\EntryLang
+
 default_config:
     default_author: "Redaktion"
     default_thumbnail: "neues_entry_fallback_image.png"
+    # TODO: Text nach *.lang verlagern
     external_url_label: "Zur Website"
```

Der Wert ist fürs Frontend, das hat erstmal nichts in der .lang zu suchen.

```suggestion
    external_url_label: "Zur Website"
```